CodeEdit/Features/WindowCommands/CodeEditCommands.swift

```diff
@@ -8,13 +8,16 @@
 import SwiftUI
 
 struct CodeEditCommands: Commands {
+    @AppSettings(\.sourceControl.general.sourceControlIsEnabled)
+    private var sourceControlIsEnabled
+
     var body: some Commands {
         MainCommands()
         FileCommands()
         ViewCommands()
         FindCommands()
         NavigateCommands()
-        SourceControlCommands()
+        if sourceControlIsEnabled { SourceControlCommands() }
```

Comment:
I don't think removing these entirely is good UX. If a user disables it accidentally, they'll just not find it at all instead of seeing a disabled state. Would it work better to use `.disabled` on the menu items here?

Reply:
That was initially my thought and I had it disabling all the other controls as you say, but I then noticed macOS Settings hiding settings when turning off the entire feature. Maybe it's a "out of sight, out of mind" thing? It is much simpler too, less checks. 🤷‍♂️

https://github.com/user-attachments/assets/147b5aeb-49ab-4041-8fef-819ce8e49d28

If you want me to I can change it back.